tests/unit/service/test_files_service.py

```diff
@@ -138,6 +138,11 @@ async def test_upload_paths_nested(
                 in mocked_upload_file_paths.call_args[1]["file_paths"]
             )
 
+            assert (
+                Path("tests/data/upload_folder_nested/meta/example.txt.meta.json")
+                in mocked_upload_file_paths.call_args[1]["file_paths"]
+            )
```

Comment:
nit nit nit: totally fine and makes sense to assert this here. I skipped this to make the test as atomic as possible and only test the intended case: "nesting files in folders and uploading them"

Reply:
(but please keep this here, all good)

Reply:
I can add it as a separate case - Added this assert due to a bug in the previous implementation